### PR TITLE
Changes total value locked into total value stake in farm page, and f…

### PR DESCRIPTION
**Pull request by geonellov**
…ix Earned column width

## Description of Changes
Changes total value locked into total value stake in farm page, and fix Earned column width

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-594)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [ ] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [x] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

**Review by tracyarciaga — APPROVED** [review]
LGTM

File: src/components/Farm/FarmPoolTable.tsx [hunk 1]
```diff
@@ -76,15 +76,15 @@ const FarmPoolTable = ({ poolsInfo, v0PoolsInfo, v1PoolsInfo, tokenPrice, select
...
+                    <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('totalValueStaked')}</TYPE.thHeader>
                   </RowFixed>
                   <RowFixed width="13%">
                     <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('stakeable')}</TYPE.thHeader>
                   </RowFixed>
                   <RowFixed width="16%">
                     <TYPE.thHeader style={{ justifySelf: 'flex-start' }}>{t('valueStaked')}</TYPE.thHeader>
                   </RowFixed>
-                  <RowFixed width="16%">
+                  <RowFixed width="20%">
```
Comment (schystz) [inline src/components/Farm/FarmPoolTable.tsx]: @geonellov width has been changed here but no further adjustment on other `RowFixed` siblings, the sum of the siblings should be 100%

File: src/components/Farm/FarmPoolCard.tsx [hunk 2]
```diff
@@ -963,7 +963,7 @@ export default function FarmPoolCard({
             <LabelText>{t('valueStaked')}</LabelText>
             <StyledTextForValue>{formatNumber(bptStakedValue, NumberFormat.usd)}</StyledTextForValue>
           </StyledRowFixed>
-          <StyledRowFixed width="16%">
+          <StyledRowFixed width="20%">
```
Comment (schystz) [inline src/components/Farm/FarmPoolCard.tsx]: Same here

**Review by schystz — COMMENTED** [review]
A couple of comments @geonellov -

File: src/components/Farm/FarmPoolTable.tsx (hunk 1 above)
Comment (geonellov) [reply src/components/Farm/FarmPoolTable.tsx]: @schystz fixed.

File: src/components/Farm/FarmPoolCard.tsx (hunk 2 above)
Comment (geonellov) [reply src/components/Farm/FarmPoolCard.tsx]: @schystz fixed.

**Review by schystz — APPROVED** [review]

**Pull request merged**